sabaki/scripts/ssh-katago-custom.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+bash_command="/nas/ucb/tony/go-attack/gtp-host/go_attack/sabaki/scripts/docker-katago-custom.sh $@"
+echo ${bash_command}
+ssh rnn -tt 'bash -l -c "'${bash_command}'"'
```

Comment:
What is this script used for?

Reply:
Ah just an old file that was previously used but not anymore since I put all the commands into Sabaki directly. Will delete.